llvm/lib/CAS/OnDiskGraphDB.cpp

```diff
@@ -67,7 +68,7 @@ static constexpr StringLiteral DataPoolTableName = "llvm.cas.data";
 static constexpr StringLiteral IndexFile = "index";
 static constexpr StringLiteral DataPoolFile = "data";
 
-static constexpr StringLiteral FilePrefix = "v7.";
+static constexpr StringLiteral FilePrefix = "v8.";
```

Comment:
In practice the version change is not necessary because I _just_ cherry-picked the "v6 -> v7" change to stable branch, but this is harmless anyway 🤷🏻‍♂️

Reply:
I left this just to be extra careful.  It's a huge pain to investigate memory errors caused by concurrent file system issues, so I prefer to define this specific case away.